src/olympia/addons/management/commands/process_addons.py

```diff
@@ -147,6 +154,12 @@ def add_arguments(self, parser):
             type=int,
             help='Only apply task to the first X addon ids.')
 
+        parser.add_argument(
+            '--distinct',
+            action='store_true',
+            dest='distinct',
+            help='Apply DISTINCT() to the query.')
```

Comment:
Instead of requiring a `--distinct` on the command line I personally would prefer forwarding this as configuration via `'kwargs': {'distinct': True},` or so so that this stays an implementation detail rather than being an argument.

Reply:
https://github.com/mozilla/addons-server/pull/11525/commits/66c04f1d5981bdd62b031fda23d70ae2b168e5c2